helms/odahu-flow-syncer/templates/syncer/sa.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.serviceAccount.create }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccount.name }}
+{{- if .Values.serviceAccount.annotations }}
+  annotations: {{ toYaml .Values.serviceAccount.annotations | nindent 4 }}
+{{- end }}
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: odahu-syncer
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - ""
+  verbs:
+  - ""
+
+---
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
Do we actually need V1beta API here?
`ClusterRoleBinding` seems to be deprecated in V1beta API in K8S 1.17+ which is going to become our base version.

Reply:
Yep it's deprecated but will be served until 1.20. Fixed in 85b8157